### PR TITLE
Natspec update for demeter protocol.

### PR DESCRIPTION
**Pull request by TechnoGeek01**

### CODE REVIEW

File: contracts/e20-farms/E20Farm.sol [hunk 1]
```diff
@@ -56,27 +59,27 @@ abstract contract E20Farm is ExpirableFarm, OperableDeposit {
...
         RewardTokenData[] memory _rwdTokenData
     ) external initializer {
-        // initialize farmToken related data
+        // initialize farmToken related data.
         farmToken = _farmToken;
         _setupFarm(_farmId, _farmStartTime, _cooldownPeriod, _rwdTokenData);
         _setupFarmExpiry(_farmStartTime, _farmRegistry);
     }
 
-    /// @notice Function is called when user transfers the NFT to the contract.
```
Comment (parv3213) [inline contracts/e20-farms/E20Farm.sol:65]: This one is better.

File: contracts/e20-farms/E20Farm.sol [hunk 2]
```diff
@@ -29,25 +29,28 @@ import {RewardTokenData} from "../Farm.sol";
...
+    /// @notice Constructor.
+    /// @param _farmStartTime - Farm start time.
+    /// @param _cooldownPeriod - Cooldown period for locked deposits in days.
     /// @dev _cooldownPeriod = 0 Disables lockup functionality for the farm.
-    /// @param _farmRegistry - Address of the Demeter Farm Registry
-    /// @param _farmToken Address of the farm token
-    /// @param _rwdTokenData - init data for reward tokens
+    /// @param _farmRegistry - Address of the Demeter Farm Registry.
+    /// @param _farmToken Address of the farm token.
+    /// @param _rwdTokenData - Init data for reward tokens.
```
Comment (parv3213) [inline contracts/e20-farms/E20Farm.sol]:
```suggestion
    /// @param _rwdTokenData - Initialize data for reward tokens.
```

File: contracts/e721-farms/E721Farm.sol [hunk 3]
```diff
@@ -39,10 +42,11 @@ abstract contract E721Farm is Farm, IERC721Receiver {
 
     /// @notice Function is called when user transfers the NFT to this farm.
     /// @param _from The address of the owner.
-    /// @param _tokenId nft Id generated by other protocol (e.g. Camelot or Uniswap).
+    /// @param _tokenId Nft Id generated by other protocol (e.g. Camelot or Uniswap).
```
Comment (parv3213) [inline contracts/e721-farms/E721Farm.sol]:
```suggestion
    /// @param _tokenId NFT Id generated by other protocol (e.g. Camelot or Uniswap).
```

File: contracts/e721-farms/camelotV2/CamelotV2Farm.sol [hunk 4]
```diff
@@ -33,26 +33,29 @@ import {OperableDeposit} from "../../features/OperableDeposit.sol";
...
+    /// @notice constructor.
+    /// @param _farmStartTime - farm start time.
+    /// @param _cooldownPeriod - Cooldown period for locked deposits in days.
     /// @dev _cooldownPeriod = 0 Disables lockup functionality for the farm.
-    /// @param _farmRegistry - Address of the Demeter Farm Registry
-    /// @param _camelotPairPool - Camelot lp pool address
-    /// @param _rwdTokenData - init data for reward tokens
+    /// @param _farmRegistry - Address of the Demeter Farm Registry.
+    /// @param _camelotPairPool - Camelot lp pool address.
+    /// @param _rwdTokenData - Init data for reward tokens.
```
Comment (parv3213) [inline contracts/e721-farms/camelotV2/CamelotV2Farm.sol]:
```suggestion
    /// @param _rwdTokenData - Initialize data for reward tokens.
```

File: contracts/utils/TokenUtils.sol [hunk 5]
```diff
@@ -7,9 +7,17 @@ import {IUniswapV3Utils} from "../e721-farms/uniswapV3/interfaces/IUniswapV3Util
 import {ICamelotV3Utils} from "../e721-farms/camelotV3/interfaces/ICamelotV3Utils.sol";
 import {ICamelotV3PoolState} from "../e721-farms/camelotV3/interfaces/ICamelotV3.sol";
```
Comment (parv3213) [inline contracts/utils/TokenUtils.sol:32]: Add Sperax Logo?

**Review by parv3213 — APPROVED** [review]

File: contracts/e20-farms/E20Farm.sol (hunk 1 above)
Comment (TechnoGeek01) [reply contracts/e20-farms/E20Farm.sol:65]: This is a E20 farm. No NFT's involved.

File: contracts/utils/TokenUtils.sol (hunk 5 above)
Comment (TechnoGeek01) [reply contracts/utils/TokenUtils.sol:32]: Done. Added demeter logo in TokenUtils.sol.
Fixed in [964912b](https://github.com/Sperax/Demeter-Protocol/pull/58/commits/964912b978d249228a789c8e6d400cdd18969d0b)

**Review by YashP16 — APPROVED** [review]
@arcantheon , @TechnoGeek01 , 
Check the natspec for newly added rewarder contracts as well

**Review by YashP16 — APPROVED** [review]

**Comment by TechnoGeek01** [comment]
Rewarder docs look good. we can merge this.

**Pull request merged**